### PR TITLE
Add `Attendance` to Developer Guide

### PR DESCRIPTION
**Pull request by andrechuakj**
What have you done in this PR?
* Add `Attendance` to DG
* Create AttendanceClassDiagram

### CODE REVIEW

File: docs/diagrams/AttendanceClassDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Class Name
+Class Remark
+}
+
+Class HiddenOutside #FFFFFF
+HiddenOutside ..> UniqueAttendanceList
+
+UniqueAttendanceList ..> "*" Attendance
+Attendance *---> "Member 1" Name
+Attendance *---> "1 Event" Name
```
Comment (dylkaw) [inline docs/diagrams/AttendanceClassDiagram.puml:24]: Should 'Member' and 'Event' be 'MemberName' and 'EventName' to be consistent with the attributes in attendance?

**Review by dylkaw — APPROVED** [review]
Just a small comment, otherwise LGTM!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -154,6 +154,42 @@ Classes used by multiple components are in the `seedu.ccacommander.commons` pack
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Attendance Model
+This section explains how the `Attendace` model is implemented and the various design consideration when
```
Comment (dylkaw) [inline docs/DeveloperGuide.md]:
```suggestion
This section explains how the `Attendance` model is implemented and the various design considerations when
```
Typo here.

**Review by dylkaw — CHANGES_REQUESTED** [review]
Just a typo that needs to be fixed, otherwise LGTM!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (andrechuakj) [reply docs/DeveloperGuide.md]: Will amend

File: docs/diagrams/AttendanceClassDiagram.puml (hunk 1 above)
Comment (andrechuakj) [reply docs/diagrams/AttendanceClassDiagram.puml:24]: As this is a class diagram, and there are no `EventName` or `MemberName` classes, I think we should keep it to `Member` and `Event`

**Review by dylkaw — APPROVED** [review]

**Review by wr1159 — APPROVED** [review]
LGTM

**Pull request merged**